ui/react-components/components/SuccessModal/SuccessModal.jsx

```diff
@@ -29,11 +29,13 @@ const SuccessModal = (props) => {
             </div>
             <div>
                 <h1 className={classNames(saveModalTitle)}>
-                    <FormattedMessage id={'APPOINTMENT_SAVE_SUCCESS_TITLE'} defaultMessage={isEdit ? 'Update Successful!' : 'Save successful'}/>
+                    <FormattedMessage id={'APPOINTMENT_SAVE_SUCCESS_TITLE'} defaultMessage={'Save successful'}/>
```

Comment:
We showing the same message for both add & edit?

Reply:
We have seperate modal in edit. So removed here.

Reply:
ok